### DISCUSSION TITLE
Using Cozie in new versions of Fitbit

### DISCUSSION THREAD

**Discussion by mateusbavaresco** [buds-lab/cozie#130] [General]
Dear Cozie team,

We want to use the Cozie app in a recent project related to human comfort in living labs. 
The current documentation shows that Cozie is stable in Fitbit Versa 3 & Sense.

I wonder if we could somehow use it also in the updated versions (e.g. Versa 4 or Sense 2). Have anyone used before?

Thank you very much,
Mateus

**Comment by mariofrei** [comment]
Hi Mateus,

Unfortunately, the answer is no. The last time I checked third-party apps were not supported on the Fitbit Versa 4 or Fitbit Sense 2. 
You'll find the sources [here](https://cozie-fitbit.app/docs/picking-a-fitbit).

Best,
Mario
